File: 6th Semester/Programming Paradigms/NP2 Exercises - Part 2/ex06.py
# – Escreva um programa que exiba a tabuada do número digitado de 0 a 10.

tabuada = int(input('Digite o número cuja tabuada deseja ser consultada: '))

for i in range(11):
    print(f'{tabuada} x {i} = {tabuada * i}')

File: 6th Semester/Programming Paradigms/NP2 Exercises - Part 2/ex06.c
// – Escreva um programa que exiba a tabuada do número digitado de 0 a 10.
#include <stdio.h>

int main()
{
    int tabuada;
    printf("Qual número deseja consultar a tabuada? ");
    scanf("%d", &tabuada);
    for (int i = 0; i <= 10; i++)
        printf("%d x %d = %d\n", tabuada, i, i * tabuada);
    return 0;
}